Mathlib/LinearAlgebra/Prod.lean

```diff
@@ -1006,7 +1006,7 @@ theorem tailings_disjoint_tunnel (f : M × N →ₗ[R] M) (i : Injective f) (n :
     apply tailing_disjoint_tunnel_succ
   · simp only [tailings_succ]
     refine' Disjoint.disjoint_sup_left_of_disjoint_sup_right _ _
-    apply tailing_disjoint_tunnel_succ
+    · apply tailing_disjoint_tunnel_succ
```

Comment:
For just reading the source, using `refine` would be quite clearer: AIUI, the `refine` creates two goals, the first of which is cdot-ed?

Reply:
Done!  And indented the second one as well.